frontend/src/pages/Projects/Project/ProjectDetails/ContactsSection/index.tsx

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
why isn't this using the ReadEditPage, I'm assuming there is significant differences, if so what are they, and should this be created as its own component.

Reply:
It's possible this could use ReadEditPage but I think it would require a lot of changes. We need to build the read/edit fields dynamically based on what is returned from the API but the ReadEditPage component handles the API request itself so we'd have to have a way to hook into that request after it is returned but before the component renders to do that processing to build the fields.